gradle-jdks/build.gradle

```diff
@@ -68,5 +69,12 @@ tasks.withType(JavaCompile).configureEach {
 
 tasks.test {
     environment("PROJECT_VERSION", project.version)
+    systemProperty 'ignoreDeprecations', 'true'
+}
+
+tasks.withType(ProcessResources.class).configureEach {
     dependsOn ':gradle-jdks-setup:fatJar'
+    from("${project(':gradle-jdks-setup').projectDir}/build/libs/gradle-jdks-setup-all-${project.version}.jar") {
```

Comment:
I think you can avoid the depends on by doing `from(':gradle-jdks-setup:fatJar')` (or `from(project(':gradle-jdks-setup').tasks.named('fatJar') if that doesn't work - this one might require an `evaluationDependsOn `gradle-jdks-setup`)